src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -2031,6 +2031,15 @@
   <data name="PreventNumericIntPtrUIntPtrBehavioralChangesConversionThrowsMessage" xml:space="preserve">
     <value>Starting with .NET 7 the explicit conversion '{0}' will throw when overflowing in a checked context. Wrap the expression with an 'unchecked' statement to restore the .NET 6 behavior.</value>
   </data>
+  <data name="UseStartsWithInsteadOfIndexOfComparisonWithZeroDescription" xml:space="preserve">
+    <value>It is both clearer and likely faster to use 'StartsWith' instead of comparing the result of 'IndexOf' to zero.</value>
```

Comment:
NIT: Doesn't seem need to have `likely` here
```suggestion
    <value>It is both clearer and faster to use 'StartsWith' instead of comparing the result of 'IndexOf' to zero.</value>
```

Reply:
fwiw, this was a copy from

https://github.com/dotnet/roslyn-analyzers/blob/79cc00251332fe690d9cbba8f367c7b21a199bc0/src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx#L1681